plugin/src/main/java/org/openrewrite/gradle/ResolveRewriteDependenciesTask.java

```diff
@@ -66,6 +66,7 @@ public Set<File> getResolvedDependencies() {
                     deps.create("org.openrewrite:rewrite-json:" + rewriteVersion),
                     deps.create("org.openrewrite:rewrite-kotlin:" + extension.getRewriteKotlinVersion()),
                     deps.create("org.openrewrite:rewrite-java:" + rewriteVersion),
+                    deps.create("org.openrewrite:rewrite-java-21:" + rewriteVersion),
```

Comment:
The first rewrite version to have rewrite-java-21 is 8.8.0.
Older rewrite versions will now fail to resolve dependencies.
Figured that's ok, but though to point it out as it failed a test.

Reply:
I agree that's probably fine. If we ever run into a case where it becomes more difficult, we can always wrap a condition around the version so as to only enable it when it's new enough.

Reply:
We could add a conditional by looking at the rewriteVersion, but [8.8.0 was released in October](https://github.com/openrewrite/rewrite/releases/tag/v8.8.0) and there's likely other incompatibilities between older rewrite versions and the latest gradle plugin version by now anyway, so I figure there's less of a need to handle this differently.

Reply:
Yep, I wasn't meaning to add it now because like you said, it's been out for a while now and there are most certainly going to be other quirks. But more just figured that I'd mention it for the hypothetical of us being on top of the Java 25 LTS more quickly.

Reply:
Agreed; your first comment only showed after I had replied with my second comment; so it wasn't as much a reply as a parallel similar thought. Thanks for the help and quick review!